src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -58,34 +69,16 @@ export const bundleJsFile = async function ({
   const targetDirectory = await tmpName()
 
   // De-duping external and ignored modules.
-  const external = [...new Set([...externalModules, ...ignoredModules])]
+  const external = [
+    ...new Set([...externalModules, ...ignoredModules, ...includedFilesToEsbuildExternals(config.includedFiles ?? [])]),
```

Comment:
Could you add some context on why we're now adding paths from `includedFiles` into esbuild's `external`?